packages/admin-stories/src/docs/components/Table/FilterBar/FilterBar.stories.mdx

```diff
@@ -0,0 +1,52 @@
+import { Meta, Story, Canvas } from "@storybook/addon-docs";
+
+<Meta title="Docs/Components/Table/Filterbar/FilterBar" />
```

Comment:
Filterbar or FilterBar? We should be consistent here. I'm rooting for FilterBar

Reply:
yes thats true :) i am also tending to FilterBar.